react-app/webpack/loaders.js

```diff
@@ -0,0 +1,123 @@
+const path = require('path');
```

Comment:
Abstracting parts of `webpack.config.js` (like plugins or loaders) is a great strategy, though you don't seem to be currently making any use of it, since (based on `package.json`) your app is still fired up by `react-scripts`.